src/components/preview/PersonalInfo.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { useState } from 'react';
+
+const PersonalInfo = props => {
+  const { personalInfo, preview } = props;
+
+  const personalInfoList = Object.keys(personalInfo).map(key => (
+    <PersonalInfoItem key={key} title={key} data={personalInfo[key]} preview={preview}></PersonalInfoItem>
+  ));
+
+  return (
+    <div>
+      <div>
+        <h1>Personal Information</h1>
+        <span>Edit</span>
+      </div>
+      <ul>{personalInfoList}</ul>
+    </div>
+  );
+};
+
+PersonalInfo.propTypes = {
+  personalInfo: PropTypes.object,
+  preview: PropTypes.bool,
+};
+
+const PersonalInfoItem = props => {
+  const { title, data, preview } = props;
+
+  const [hidden, setHidden] = useState(0);
+
+  return data ? (
+    <li>
+      <div>
+        <div>
+          <span>Your {title.charAt(0).toUpperCase() + title.slice(1)}</span>
+          <p>{data.data}</p>
+          {hidden ? <span>Hidden</span> : <></>}
+        </div>
+        {preview ? (
```

Comment:
you can do conditional rendering like:
!preview && <button />

Reply:
[form-preview c718227] form-preview:#2 - add conditional rendering